### PR TITLE
[core] add OpenShift::portForward method

### PR DESCRIPTION
**Pull request by dsimansk**
Simple `PodPortForward` impl without `oc` binary.

### CODE REVIEW

File: core/src/main/java/cz/xtf/core/openshift/PodPortForward.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	private String podName;
+
+	private LocalPortForward connection;
+
+	public PodPortForward(OpenShift openShift, Pod pod) {
+		this.openShift = openShift;
+		this.podName = pod.getMetadata().getName();
+	}
+
+	public PodPortForward(OpenShift openShift, String dcName) {
```
Comment (Crumby) [inline core/src/main/java/cz/xtf/core/openshift/PodPortForward.java]: Could we switch constructors order? I consider it more readable if called constructor (method) is bellow instead above.

File: core/src/main/java/cz/xtf/core/openshift/PodPortForward.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package cz.xtf.core.openshift;
+
+import io.fabric8.kubernetes.api.model.Pod;
+import io.fabric8.kubernetes.client.LocalPortForward;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class PodPortForward implements AutoCloseable {
+
```
Comment (Crumby) [inline core/src/main/java/cz/xtf/core/openshift/PodPortForward.java]: Could we delete this empty line? Just to keep it consistent with other classes.

**Comment by Crumby** [comment]
What is an actual advantage of this class, in the end usage meaning. Autoclosability and accessibility with just dcName?

File: core/src/main/java/cz/xtf/core/openshift/PodPortForward.java (hunk 2 above)
Comment (dsimansk) [reply core/src/main/java/cz/xtf/core/openshift/PodPortForward.java]: Sure, we probably need a codestyle check then :)

**Comment by dsimansk** [comment]
@Crumby the motivation is to keep AutoCloseable that we used to be provided via `PodService`. There's no additional value actually. Maybe, we could extend it a bit more to reconnect on failure? Would that make more sense?

File: core/src/main/java/cz/xtf/core/openshift/PodPortForward.java (hunk 2 above)
Comment (Crumby) [reply core/src/main/java/cz/xtf/core/openshift/PodPortForward.java]: Created an issue https://github.com/xtf-cz/xtf/issues/104 for that.

**Pull request merged**